client-docs/README.md

```diff
@@ -76,10 +79,13 @@ type ProjectInfo = {
 
 type ProjectRole = "creator" | "coordinator" | "member";
 
-type Invite = {
-  projectId: ProjectId;
+type PendingInvite = {
+  inviteId: string;
   projectName: string;
-  role: ProjectRole;
+  roleName: string;
+  roleDescription: string;
+  invitorName?: string;
+  timeOfInvite: Date;
```

Comment:
@EvanHahn I think the backend should track the time. The reason is there is a possibility that an invitee opens mapeo, is part of the network and gets sent a invite, but react has not been initialized, so there would be no way for the front end to know which invites came first

Reply:
We can do that, but what if it just returns them in the order they were received?